exec/transport-spire

```diff
@@ -18,7 +18,8 @@ case "$1" in
 				stark_s:.starkSignature.s,
 				stark_pk:.starkSignature.publicKey
 			},
-			trace: .sources
+			trace: .sources,
+			version: .version
```

Comment:
@teghnet this is already present in omnia:

https://github.com/chronicleprotocol/omnia/blob/b426364ca74a8b317dc208e38cb5089cdbce21e2/lib/feed.sh#L103

and it uses the `OMNIA_VERSION` env var. is it even necessary then to pass anything else? can't we infer the versions of components like `oracles`, `setzer` from `OMNIA_VERSION`?

Reply:
We sometimes update configs only and they reside in [oracles/systemd](https://github.com/chronicleprotocol/oracles/tree/master/systemd).

We'd like to know if feeds updated configs as well.

Reply:
I see. hm, in that case then I think we should change `version` to `versions` and make it a map, i.e.

```
versions: {
   "omnia": "0.1.1",
   "spire": "0.1.1.",
   ... etc ...
}
```

Reply:
The thing is that `ORACLE_VERSION` precisely determines all other versions.